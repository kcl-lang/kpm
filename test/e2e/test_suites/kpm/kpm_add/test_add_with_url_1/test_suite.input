kpm add https://github.com/KusionStack/catalog.git?commit=3891e96