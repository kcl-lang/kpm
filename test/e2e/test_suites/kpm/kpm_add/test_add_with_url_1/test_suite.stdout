add dependency 'catalog:3891e96' successfully